packages/react/src/components/UserProvider.tsx

```diff
@@ -92,6 +92,7 @@ export const UserProvider = (props: Props) => {
       setUser(user);
       // Set up auto token refresh
       if (autoRefreshToken) {
+        // TODO: the user.exp is a bug that is currently working like a feature
```

Comment:
can you expand on that?

Reply:
I've opened a issue explaining why this is a bug https://github.com/supabase-community/auth-helpers/issues/134